CONTRIBUTING.md

```diff
@@ -120,67 +118,22 @@ The scope is optional but should be included when possible and refer to a module
 
 The subject contains succinct description of the change:
 
-- use the imperative, present tense: "change" not "changed" nor "changes"
-- don't capitalize first letter
-- no dot (.) at the end
+- Use the imperative present tense (e.g., "change" not "changed" nor "changes")
+- Don't capitalize first letter
+- No dot (.) at the end
 
-#### Body
+#### Body (optional)
 
-Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes" The body should include the motivation for the change and contrast this with previous behavior.
+Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes". The body should include the motivation for the change and contrast this with previous behavior.
 
-#### Footer
+#### Footer (optional)
 
 The footer should contain any information about **Breaking Changes** and is also the place to reference GitHub issues that this commit **Closes**.
 
 The last line of commits introducing breaking changes should be in the form `BREAKING CHANGE: <desc>`
 
 Breaking changes should also add an exclamation mark `!` after the type/scope (e.g. `refactor(rt)!: drop support for Android API < 20`)
 
-### Automated PR checks
-
-A number of automated workflows run when a PR is submitted. Generally speaking, each of these must pass before the PR is
-allowed to be merged. If your PR fails one of the checks, please attempt to address the problem and push a new commit to
-the PR. If you need help understanding or resolving a PR check failure, please reach out via a PR comment or a GitHub
-discussion. Please file a new issue if you believe there's a pre-existing bug in a PR check.
-
-#### Lint
```

Comment:
I think this information about linting is helpful for new contributors -- personally I did not know about `ktlint` before joining.

I see we have the same section about linting in [aws-sdk-kotlin](https://github.com/awslabs/aws-sdk-kotlin/blob/main/CONTRIBUTING.md#lint), so was this removed mainly for deduplication?

Reply:
Whoops, this was unintentionally removed. It should absolutely still be here.